### ISSUE TITLE
Past, Present and Future of Open Science (Emergent session): Machine learning in neuroimaging: what do we agree on and where are we headed?

### ISSUE THREAD

**Issue by jsheunis** [ohbm/osr2020#69]
# Machine learning in neuroimaging: what do we agree on and where are we headed?
***By Sanmi Koyejo, University of Illinois at Urbana-Champaign***

- Theme: Past, Present and Future of Open Science
- Format: Emergent session

## Abstract
Has machine learning for neuroimaging lived up the hype? Will deep learning revolutionize neuroimaging? Will the use of machine learning and neuroimaging change either our understanding of human brain function or clinical care? Join us to discuss the past, present, and future of machine learning in neuroimaging. Our discussion will outline the promise of machine learning in neuroimaging, where it has been successful, and where the excitement has led to disappointment. We will also discuss current best practices and some current exciting applications. Building on this history and experiences, we will look forward to the future of machine learning in neuroimaging, will discuss emerging opportunities for impact, and will ask what is needed from the neuroimaging community.

## Useful Links
[]()
[]()

Tagging @sanmik

**Comment by Starborn** [comment]
Yes, I am interested in this topic, loosely related to my short talk, trying to figure out how to become a participant

**Comment by Starborn** [comment]
Please share notes/slides and any input you may like. My take is that there is some high level knowledge representation missing which is what I am working on

**Comment by complexbrains** [comment]
This event has been **scheduled to be run on 02.07.2020, 19:00- 20:00 GMT**

For more information, please go to  https://ohbm.github.io/osr2020/schedule/emea

**Comment by Starborn** [comment]
Greetings, please share recordings/notes? I am sorry I was not around
really interested in this topic, @sanmi how do we connect?
